inventory/production.yml

```diff
@@ -1,11 +1,9 @@
 ---
 all:
-  vars:
-    ansible_python_interpreter: auto
   hosts:
     obs:
       ansible_connection: ssh
       # configured host in your ssh config
       ansible_host: obs
-      apache_sysconfig_regex_unset: 'APACHE_SERVER_FLAGS="STATUS"' 
+      apache_sysconfig_regex_unset: 'APACHE_SERVER_FLAGS="STATUS"'
```

Comment:
is that change related with the change with python2/python3?

Reply:
Nop. Should I separate it in another commit?

Reply:
Ops, you edited your comment, and added another one. 

> Did you test manually if it's working?

Yes, sure.

Reply:
> Nop. Should I separate it in another commit?

No not necessary, I just wanted to know if it was something commited by accident

Reply:
> is that change related with the change with python2/python3?

I created another commit.

Reply:
> > Nop. Should I separate it in another commit?
> 
> No not necessary, I just wanted to know if it was something commited by accident

Ups, late.